.github/CODEOWNERS

```diff
@@ -1849,6 +1849,9 @@ packages/kbn-monaco/src/esql @elastic/kibana-esql
 /config/serverless.es.yml @elastic/kibana-core @elastic/kibana-security
 /config/serverless.oblt.yml @elastic/kibana-core @elastic/kibana-security
 /config/serverless.security.yml @elastic/kibana-core @elastic/kibana-security
+/config/serverless.security.search_ai_lake.yml @elastic/kibana-core @elastic/kibana-security
+/config/serverless.security.essentials.yml @elastic/kibana-core @elastic/kibana-security
+/config/serverless.security.complete.yml @elastic/kibana-core @elastic/kibana-security
```

Comment:
nit-Q: Should Core own these?

Reply:
Thanks for asking @afharo I'm not sure actually. Given that these are security specific configs, I guess @elastic/security-solution should own these? WDYT @YulNaumenko?

Reply:
I agree with @ashokaditya, it would be great if this is possible.

Reply:
Thanks @YulNaumenko, updated in 74ae4bc66bec1aa5f02b17494fbd998bfbe7bd95